src/client/datascience/commands/exportCommands.ts

```diff
@@ -63,9 +63,10 @@ export class ExportCommands implements IDisposable {
         this.disposables.push(disposable);
     }
 
-    private async nativeNotebookExport(uri?: Uri) {
-        const editor = uri
-            ? this.notebookProvider.editors.find((item) => this.fs.arePathsSame(item.file, uri))
+    private async nativeNotebookExport(uri?: Uri | { notebookEditor: { notebookUri: Uri } }) {
+        const notebookUri = uri !== undefined && 'notebookEditor' in uri ? uri.notebookEditor.notebookUri : uri;
```

Comment:
```suggestion
        const notebookUri = isUri(uri) ? uri : uri.notebookEditor.notebookUri;
```

Reply:
You can use the helper `isUri` instead, you'd need to import that into this file.